index.css

```
@@ -0,0 +1,725 @@
+body
+{
+    font-family: Arial, Helvetica, sans-serif;
+    box-sizing: border-box;
+    margin: 0;
+}
+
+.main-input
+{
+    display: none;
+}
+
+.header
+{
+    width: 1200px;
+    margin: 15px auto;
+    display: grid;
+    grid-template-columns: 110px 1fr 110px 200px;
+    grid-template-rows: auto;
+}
+
+.header__cities
+{
+    margin: 10px 0 0;
+    color: #9a9a9a;
+    font-size: 14px;
+    display: flex;
+}
+
+.header__cities-icon

```

```suggestion
.cities__icon
```
?

Потому что сейчас не по БЕМ.